lib/partial/transmuxer.js

```diff
@@ -191,7 +192,7 @@ var aacPipeline = function(options) {
 
     pipeline.tracks.audio = pipeline.tracks.audio || {
       timelineStartInfo: {
-        baseMediaDecodeTime: !options.keepOriginalTimestamps ? options.baseMediaDecodeTime : 0
+        baseMediaDecodeTime: options.baseMediaDecodeTime
```

Comment:
Do we still want to allow setting of `baseMediaDecodeTime` if `keepOriginalTimestamps` is true?

Reply:
We do in the main transmuxer, so we do in the partial as well.